python/avogadro/cjson.py

```diff
@@ -0,0 +1,41 @@
+"""
+/******************************************************************************
+  This source file is part of the Avogadro project.
+  This source code is released under the New BSD License, (the "License").
+******************************************************************************/
+"""
+import json
+class Cjson:
+    """
+    This Class is intended to read Cjson files
+    with python libraries and perform certain
+    methods on files and convert them back to Cjson
+    files as required
+    """
+    def __init__(self):
+        pass
+    def __open_file(self, filePath):
+        '''Use to read CJson formats by converting them to python dictionaries'''
+        with open(filePath, 'r') as cjsonFile:
+            py_dict_data = json.load(cjsonFile)
+            return py_dict_data
+    def __to_cjson(self, cjson_dict_file):
+        '''It converts python dictionaries to CJson format'''
+        cjsonData = json.dumps(cjson_dict_file, indent=4)
+        return (cjsonData)
+    def get_atoms_coords(self,filePath):
```

Comment:
I think we want both a method to get an array of coordinates and set the data["atoms"]["coords"]["3d"] from an array of (-1, 3) coordinates.

If you want to add this method (which returns coordinates + atomic number), I'd want to also see:
- get_atoms_coordinates() - this method
- set_atoms_coordinates() - setting the data[] from an array of coordinates and elements
- get_elements() - simply returning the elements from `data["atoms"]["elements"]["number"]`
- set_elements() - setting the elements
- get_coordinates() - just return the coordinates
- set_coordinates() - just set the coordinates

Reply:
Ok, I will add all these functions shortly.

Reply:
What will be the format of param or file for set_atom_coordinates @ghutchis 
only array of coords will be given or array of coords + atomic number

Reply:
@ghutchis can you please help me with the format for input array of coordinates and elements for set_atoms_coordinates.